### PR TITLE
Updated extconf for termux installation fix

### PR DESCRIPTION
**Pull request by Auxilus**

### CODE REVIEW

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by Auxilus** [comment]
I signed it!

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

**Comment by grpc-testing** [comment]
Thanks for your pull request. The automated tests will run as soon as one of the admins verifies this change is ok for us to run on our infrastructure.

File: src/ruby/ext/grpc/extconf.rb [hunk 1]
```diff
@@ -63,19 +63,10 @@
 ENV['CFLAGS'] = '-DGPR_BACKWARDS_COMPATIBILITY_MODE'
 
 output_dir = File.expand_path(RbConfig::CONFIG['topdir'])
-grpc_lib_dir = File.join(output_dir, 'libs', grpc_config)
 ENV['BUILDDIR'] = output_dir
 
-unless windows
```
Comment (timwr) [inline src/ruby/ext/grpc/extconf.rb]: are we sure it's OK to remove this?
how about
```
if termux #check an env variable perhaps?
  grpc_lib_dir = ENV['PREFIX'] + '/lib'
else if not windows
  ...
end
```

Comment (apolcyn) [inline src/ruby/ext/grpc/extconf.rb]: This won't work because it breaks the windows build (which builds the core library in a  separate, earlier step

**Comment by Auxilus** [comment]
@apolcyn sure, I'll work on that

**Comment by Auxilus** [comment]
See https://github.com/grpc/grpc/pull/12752

Im closing this PR..

**Pull request closed without merging**